src/styles/base.js

```diff
@@ -107,6 +107,7 @@ export default theme => ({
         // lineHeight: '14px',
         fontSize: theme.ARROW_FONT_SIZE,
         marginRight: theme.ARROW_MARGIN_RIGHT,
+        transition: `transform ${theme.ARROW_ANIMATION_TIME} ease 0s`,
```

Comment:
Thanks! I should have added this to theming params when I changed arrows expansion to rotation but didn't get to it.

But I still intended to have no animation by default. Can we add transition *only if* animation time is >0?

Reply:
I made sure it doesn't animate when animation time is 0ms :)

Reply:
Sounds good, but could you make `0` as default? And don't include transition prop you don't specify animation time. Like ```...(theme.ARROW_ANIMATION_TIME && { transition: `...` }),```

Reply:
done.